--- poppler-0.62.0/glib/reference/CMakeLists.txt.orig	2018-02-12 22:14:30.418990148 +0100
+++ poppler-0.62.0/glib/reference/CMakeLists.txt	2018-02-12 22:15:59.992430316 +0100
@@ -8,5 +8,5 @@
 add_custom_target(glib-docs ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/glib-docs-build.stamp)
 
 install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/html/
-  DESTINATION "${CMAKE_INSTALL_DATADIR}/gtk-doc/html/poppler"
+  DESTINATION "${CMAKE_INSTALL_DOCDIR}/gtk-doc/html/poppler"
 )
