--- poppler-0.5.2/m4/qt.m4.orig	2006-05-02 06:03:46.000000000 +0200
+++ poppler-0.5.2/m4/qt.m4	2006-05-27 02:43:01.784504500 +0200
@@ -7,8 +7,8 @@
 AC_DEFUN([POPPLER_FIND_QT],
 [
 dnl Search paths for Qt 
-qt_incdirs="$QTINC /usr/local/qt/include /usr/include/qt /usr/include /usr/X11R6/include/X11/qt /usr/X11R6/include/qt /usr/lib/qt3/include /usr/lib/qt/include /usr/share/qt3/include"
-qt_libdirs="$QTLIB /usr/local/qt/lib /usr/lib/qt /usr/lib /usr/X11R6/lib/X11/qt /usr/X11R6/lib/qt /usr/lib/qt3/lib /usr/lib/qt/lib /usr/share/qt3/lib"
+qt_incdirs="/usr/include/qt"
+qt_libdirs="/usr/lib64 /usr/lib"
 if test -n "$QTDIR" ; then
     qt_incdirs="$QTDIR/include $qt_incdirs"
     qt_libdirs="$QTDIR/lib $qt_libdirs"
