--- poppler-0.62.0/gtkdoc.py.orig	2018-02-14 00:01:33.698649806 +0100
+++ poppler-0.62.0/gtkdoc.py	2018-02-14 00:02:06.508409173 +0100
@@ -319,9 +319,9 @@
             ldflags = ' "-L%s" %s ' % (self.library_path, additional_ldflags) + ldflags
             current_ld_library_path = env.get('LD_LIBRARY_PATH')
             if current_ld_library_path:
-                env['RUN'] = 'LD_LIBRARY_PATH="%s:%s" ' % (self.library_path, current_ld_library_path)
+                env['LIBRARY_PATH'] = 'LD_LIBRARY_PATH="%s:%s" ' % (self.library_path, current_ld_library_path)
             else:
-                env['RUN'] = 'LD_LIBRARY_PATH="%s" ' % self.library_path
+                env['LIBRARY_PATH'] = 'LD_LIBRARY_PATH="%s" ' % self.library_path
 
         if ldflags:
             env['LDFLAGS'] = '%s %s' % (ldflags, env.get('LDFLAGS', ''))
