--- poppler-0.8.0/qt4/src/Makefile.am.orig	2008-04-01 12:59:36.554743230 +0200
+++ poppler-0.8.0/qt4/src/Makefile.am	2008-04-01 12:59:56.053403130 +0200
@@ -63,7 +63,7 @@
 
 # This rule lets GNU make create any *.moc from the equivalent *.h
 .h.moc:
-	moc -i $< -o $@
+	moc-qt4 -i $< -o $@
 
 poppler-optcontent.lo: poppler-optcontent.moc
 
--- poppler-0.8.0/qt4/demos/Makefile.am.orig	2008-04-01 13:05:59.528422319 +0200
+++ poppler-0.8.0/qt4/demos/Makefile.am	2008-04-01 13:06:09.527735090 +0200
@@ -16,7 +16,7 @@
 
 # This rule lets GNU make create any *.moc from the equivalent *.h
 .h.moc:
-	moc -i $< -o $@
+	moc-qt4 -i $< -o $@
 
 noinst_PROGRAMS = poppler_qt4viewer
 
